### PR TITLE
Set default port for running locally

### PR DESCRIPTION
**Pull request by zachmargolis**
**Why**: Works better with our CDN

this is what we have [in our config](https://github.com/18F/identity-devops/blob/e4818146d278bdc297901cf2d17d2da6362d28c6/terraform/app/bucket-public-reporting-data.tf#L73)

### CODE REVIEW

**Review by aduth — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -7,11 +7,11 @@
   "scripts": {
     "build": "vite build",
     "postbuild": "npm run generate-routes && cp robots.txt _site/",
-    "dev": "vite",
+    "dev": "vite --port ${PORT:-3000}",
     "generate-routes": "node -r esbuild-register bin/generate-routes.js",
     "pages": "npm run build -- --base=${BASEURL:-/}",
     "prettier:fix": "npm run test:lint -- --fix",
-    "serve": "vite preview",
+    "serve": "vite preview --port ${PORT:-3000}",
```
[the remark this answers is not in the document]
Comment (zachmargolis) [reply package.json:14]: happy to do either, should we ditch both `dev` and `serve` in favor of `start`?

[the remark this answers is not in the document]
Comment (zachmargolis) [reply package.json]: Updated in fc2c0f2808ce983ee2072c443989a6b097f8e7e9

[the remark this answers is not in the document]
Comment (aduth) [reply package.json]: That works 👍 I haven't needed to use `npm run serve` 'til now, so happy to drop it for now until we find use for it.

**Pull request merged**